openseadragon.module

```diff
@@ -79,6 +79,9 @@ function template_preprocess_openseadragon_formatter(&$variables) {
         'id' => $openseadragon_viewer_id,
         'prefixUrl' => 'https://cdnjs.cloudflare.com/ajax/libs/openseadragon/2.4.2/images/',
         'tileSources' => $tile_sources,
+        'loadTilesWithAjax' => TRUE,
+        'ajaxWithCredentials' => TRUE,
+        'ajaxHeaders' => ["Authorization" => "Bearer " . \Drupal::service('jwt.authentication.jwt')->generateToken()],
```

Comment:
Cache tags/contexts would have to be added for this to avoid using the results for all users... and does it make sense to be generating tokens for anonymous users, if it's going to trash the cache?

Reply:
Good catch @adam-vessey .  I'll give it the caching treatment for users.  

If anonymous having a token breaks caching, I can tell it not to add a token for anon and then adjust the custom delegate appropriately.